### PR TITLE
Swagger

### PR DESCRIPTION
**Pull request by sebas97-stack**
## Summary
- Config swagger
- add swagger controller user
- add swagger model user
## Trello Card
https://trello.com/c/AiF1aOfV

### CODE REVIEW

File: src/main/java/wolox/training/controller/UserController.java [hunk 1]
```diff
@@ -55,6 +61,12 @@ public UserController(UserRepository userRepository, BookRepository bookReposito
      * @throws UserNotFoundException if there is no user associated with that username
      */
     @GetMapping("/username")
+    @ApiOperation(value = "Giving an username, return the user", response = User.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved user"),
+            @ApiResponse(code = 400, message = "Something went wrong"),
+            @ApiResponse(code = 404, message = "The resource you were trying to reach is not found"),
+            @ApiResponse(code = 500, message = "Internal Server Error")})
```
Comment (martinremis) [inline src/main/java/wolox/training/controller/UserController.java]: Extract those string to constants. I should be a different file, accessible by every controller

**Review by martinremis — CHANGES_REQUESTED** [review]
There is no Swagger docs for book controller?

File: src/main/java/wolox/training/config/SwaggerConfig.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                .select()
+                .apis(RequestHandlerSelectors.any())
+                .paths(PathSelectors.any())
+                .build()
+                .apiInfo(apiEndPointsInfo());
+    }
+
+    private ApiInfo apiEndPointsInfo() {
+        return new ApiInfoBuilder()
+                .title("Training API")
```
Comment (AleFalcon) [inline src/main/java/wolox/training/config/SwaggerConfig.java]: These constants could be taken to a file of constants, as if there is any change in the future, there is a single place to touch the constants

File: src/main/java/wolox/training/constants/MessageSwagger.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package wolox.training.constants;
+
+public interface MessageSwagger {
+
+    String SUCCESS_GET_USER = "Successfully retrieved user";
```
Comment (AleFalcon) [inline src/main/java/wolox/training/constants/MessageSwagger.java]: This constants could be "static final"

**Review by AleFalcon — CHANGES_REQUESTED** [review]

**Review by AleFalcon — APPROVED** [review]
Great Job

**Pull request merged**